BP/history/_BP_notes.info

```diff
@@ -0,0 +1,59 @@
+Move Bodh Gaya holy site to Bodh Gaya proper and move Mahabodhi Temple to it as well.
+
+Do Hispania after merging in Abian36's changes.
```

Comment:
Is Hispania done now or not? This night need an update.

Reply:
I've merged in Abian's changes; forgot to remove that line.